frankenstein/consul_client.go

```diff
@@ -205,3 +205,42 @@ func (c *consulClient) WatchPrefix(prefix string, factory func() interface{}, do
 		}
 	}
 }
+
+func (c *consulClient) PutBytes(key string, buf []byte) error {
+	_, err := c.kv.Put(&consul.KVPair{
+		Key:   key,
+		Value: buf,
+	}, &consul.WriteOptions{})
+	return err
+}
+
+type prefixedConsulClient struct {
+	prefix string
+	ConsulClient
```

Comment:
In this case I wouldn't embed, but rather make it a private field.  That way we'll get compiler errors if someone adds a new method to ConsulClient but not to this.